opencraft/settings.py

```diff
@@ -0,0 +1,340 @@
+TIME_ZONE = 'UTC'
+USE_I18N = True
+USE_L10N = True
+USE_TZ = True
+
+
+# Static files (CSS, JavaScript, Images) ######################################
+
+# https://docs.djangoproject.com/en/1.8/howto/static-files/
+
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    # other finders..
+    'compressor.finders.CompressorFinder',
+)
+
+STATICFILES_DIRS = (
+    root('static'),
+)
+
+STATIC_ROOT = root('build/static')
+STATIC_URL = '/static/'
+
+COMPRESS_PRECOMPILERS = (
+    ('text/x-scss', 'django_libsass.SassCompiler'),
+)
+
+
+# Django-extensions ###########################################################
+
+SHELL_PLUS = "ipython"
+RUNSERVERPLUS_SERVER_ADDRESS_PORT = '127.0.0.1:5000'
+
+
+# Grappelli ###################################################################
+
+GRAPPELLI_ADMIN_TITLE = 'OpenCraft'
+GRAPPELLI_SWITCH_USER = True
+
+
+# REST framework ##############################################################
+
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': [
+        'rest_framework.permissions.DjangoModelPermissions',
+    ],
+}
+
+
+# Redis cache & locking #######################################################
+
+REDIS_URL = env('REDIS_URL', default='redis://localhost:6379/')
+REDIS_URL_OBJ = urlparse(REDIS_URL)
+
+CACHES = {
+    'default': {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': REDIS_URL,
+        'OPTIONS': {
+            'CLIENT_CLASS': 'django_redis.client.DefaultClient',
+        }
+    }
+}
+
+
+# Huey (redis task queue) #####################################################
+
+HUEY = {
+    'backend': 'huey.backends.redis_backend',
+    'name': 'opencraft',
+    'connection': {
+        'host': REDIS_URL_OBJ.hostname,
+        'port': REDIS_URL_OBJ.port,
+        'password': REDIS_URL_OBJ.password,
+    },
+    'always_eager': env.bool('HUEY_ALWAYS_EAGER', default=False),
+
+    # Options to pass into the consumer when running ``manage.py run_huey``
+    'consumer_options': {'workers': 1, 'loglevel': logging.DEBUG},
+}
+
+
+# SwampDragon (websocket) #####################################################
+
+SWAMP_DRAGON_REDIS_HOST = REDIS_URL_OBJ.hostname
+SWAMP_DRAGON_REDIS_PORT = REDIS_URL_OBJ.port
+SWAMP_DRAGON_CONNECTION = ('swampdragon.connections.sockjs_connection.DjangoSubscriberConnection', '/data')
+DRAGON_URL = env('DRAGON_URL', default='http://localhost:2001/')
+
+
+# OpenStack ###################################################################
+
+OPENSTACK_USER = env('OPENSTACK_USER')
+OPENSTACK_PASSWORD = env('OPENSTACK_PASSWORD')
+OPENSTACK_TENANT = env('OPENSTACK_TENANT')
+OPENSTACK_AUTH_URL = env('OPENSTACK_AUTH_URL')
+OPENSTACK_REGION = env('OPENSTACK_REGION')
+
+OPENSTACK_SANDBOX_FLAVOR = env.json('OPENSTACK_SANDBOX_FLAVOR', default={"ram": 4096, "disk": 40})
+OPENSTACK_SANDBOX_BASE_IMAGE = env.json('OPENSTACK_SANDBOX_BASE_IMAGE', default={"name": "Ubuntu 12.04"})
```

Comment:
@dachary Nice, thanks! That's a clean way to get rid of that RunAbove motd header ;p